ampalibe/messenger.py

```diff
@@ -145,13 +145,10 @@ def send_quick_reply(self, dest_id, quick_rep, text, next=False):
             https://developers.facebook.com/docs/messenger-platform/send-messages/quick-replies
         """
 
-        for i in range(len(quick_rep[:12])):
-            if isinstance(quick_rep[i], QuickReply):
-                quick_rep[i] = quick_rep[i].value
-            if quick_rep[i].get("payload"):
-                quick_rep[i]["payload"] = Payload.trt_payload_out(
-                    quick_rep[i]["payload"]
-                )
+        quick_rep = [
+            quick_rep[i].value if isinstance(quick_rep[i], QuickReply) else quick_rep[i]
+            for i in range(len(quick_rep))
```

Comment:
Remettre juste qucik_rep in quick_rep 

Plus de besoin d avoir l index

Reply:
Oui c'est testé,  j'ai un peu modifié la partie de condition sur la lo,gueur du quick reply